src/main/java/com/google/jenkins/flakyTestHandler/plugin/FlakyTestResultAction.java

```diff
@@ -75,7 +75,7 @@
    */
   public FlakyTestResultAction(AbstractBuild build, BuildListener listener) {
     this.build = build;
-    AbstractTestResultAction action = build.getTestResultAction();
+    AbstractTestResultAction action = build.getAction(AbstractTestResultAction.class);
```

Comment:
I was not aware that having multiple `AbstractTestResultAction`s on one build was even legal. Anyway, to answer your question: no; but neither did the original code, so that is not relevant to this PR, which is just to restore functionality.